### PR TITLE
feat: display name support

### PR DESCRIPTION
**Pull request by prwozny**

### CODE REVIEW

File: src/pages/StatsTracking.tsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+interface StatsTrackingProps {
+  surveyUnits: SurveyUnit[];
+}
+
+/**
+ * @param {SurveyUnit[]} surveyUnits
+ */
+export function StatsTracking({ surveyUnits }: Readonly<StatsTrackingProps>) {
+  const [sortDirection, setSortDirection] = useState('');
+  const handleSortChange = (direction: React.SetStateAction<string>) => {
```
Comment (EmmanuelDemey) [inline src/pages/StatsTracking.tsx:19]: Do you need to add the extra React before SetStateAction?

File: src/pages/StatsTracking.tsx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+interface StatsTrackingProps {
+  surveyUnits: SurveyUnit[];
+}
+
+/**
+ * @param {SurveyUnit[]} surveyUnits
+ */
+export function StatsTracking({ surveyUnits }: Readonly<StatsTrackingProps>) {
+  const [sortDirection, setSortDirection] = useState('');
```
Comment (EmmanuelDemey) [inline src/pages/StatsTracking.tsx:18]: We can maybe type correctly this variable.? It can accept only defined list of string values.

**Review by EmmanuelDemey — CHANGES_REQUESTED** [review]

File: src/i18n/trackingMessage.js [hunk 3]
```diff
@@ -29,6 +29,11 @@ const trackingMessage = {
     en: 'Lastname/FirstName',
     sq: 'Mbiemri/Emri',
   },
+  trackingSearchField: {
+    fr: 'Nom/Prénom, Id, Id Métier',
+    en: 'Lastname/FirstName, Id, Nusiness Id',
```
Comment (SimonDmz) [inline src/i18n/trackingMessage.js:34]: typo Business

File: src/i18n/trackingMessage.js [hunk 4]
```diff
@@ -29,6 +29,11 @@ const trackingMessage = {
...
+    sq: 'Mbiemri/Emri',
```
Comment (SimonDmz) [inline src/i18n/trackingMessage.js:35]:
```suggestion
    sq: 'Mbiemri/Emri, id, id i njësisë',
```
🇦🇱 ✊

File: src/ui/SurveyCard.tsx [hunk 5]
```diff
@@ -79,7 +85,7 @@ export function SurveyCard({ surveyUnit, locked = false }) {
               {locked && <LockIcon color="iconLock" />}
             </Row>
             <Typography variant="s" color="textHint" as="div">
-              #{surveyUnit.id}
+              #{surveyUnit.displayName ? surveyUnit.displayName : surveyUnit.id}
```
Comment (SimonDmz) [inline src/ui/SurveyCard.tsx:88]:
```suggestion
              #{surveyUnit.displayName ?? surveyUnit.id}
```

File: src/pages/TableTracking.tsx [hunk 6]
```diff
@@ -446,6 +261,7 @@ function SurveyUnitRow({ surveyUnit }) {
               <Typography as="span" variant="s" color="textPrimary">
                 {findContactAttemptValueByType(lastContact.status)}
                 <br />
+                eeeee
```
Comment (SimonDmz) [inline src/pages/TableTracking.tsx:264]: ??? => to remove

File: src/pages/TableTracking.tsx [hunk 7]
```diff
@@ -429,9 +242,11 @@ function SurveyUnitRow({ surveyUnit }) {
       <TableRow>
         <TableCell align="center">
           {isActive ? (
-            <Link to={`/survey-unit/${surveyUnit.id}/details`}>#{surveyUnit.id}</Link>
+            <Link to={`/survey-unit/${surveyUnit.id}/details`}>
+              #{surveyUnit.displayName ? surveyUnit.displayName : surveyUnit.id}
+            </Link>
           ) : (
-            `#${surveyUnit.id}`
+            `#${surveyUnit.displayName ? surveyUnit.displayName : surveyUnit.id}`
```
Comment (SimonDmz) [inline src/pages/TableTracking.tsx:249]:
```suggestion
            `#${surveyUnit.displayName ?? surveyUnit.id}`
```

File: src/pages/TableTracking.tsx [hunk 8]
```diff
@@ -429,9 +242,11 @@ function SurveyUnitRow({ surveyUnit }) {
       <TableRow>
         <TableCell align="center">
           {isActive ? (
-            <Link to={`/survey-unit/${surveyUnit.id}/details`}>#{surveyUnit.id}</Link>
+            <Link to={`/survey-unit/${surveyUnit.id}/details`}>
+              #{surveyUnit.displayName ? surveyUnit.displayName : surveyUnit.id}
```
Comment (SimonDmz) [inline src/pages/TableTracking.tsx:246]:
```suggestion
              #{surveyUnit.displayName ?? surveyUnit.id}
```

**Review by SimonDmz — CHANGES_REQUESTED** [review]
change the version in package.json too 😉

**Review by SimonDmz — APPROVED** [review]

**Pull request merged**